ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_HasType_User()
+        {
+            //Arrange
+            User _user = new User("Tommy");
+            string title = null;
+            //Act
+            Article _article = new Article(_user);
+            //Assert
+            Assert.Null(title);
+            Assert.IsType<User>(_article.Autor);
```

Comment:
1. The rows #16 and #20 should be deleted as they do nothing (here you just like test the Microsoft .Net Framework and CLR instead of the program code).
2. Do not use checking the type of anything until the action method(s) can return the result with different types (usual it is in method overrides). Here returned type is predefined - the `Autor` property always return the type `User` even if null (an instance of `User` class or casted to `User` null value). You can test that value is not null instead.
```suggestion
            //Arrange
            User _user = new User("Tommy");
            //Act
            Article _article = new Article(_user);
            //Assert
            Assert.NotNull(_article.Autor);
```